R/servers.R

```diff
@@ -229,21 +218,20 @@ removeServer <- function(name) {
 serverInfo <- function(name) {
   check_string(name)
 
-  # there's no config file for Posit's hosted offerings
   if (identical(name, "shinyapps.io")) {
-    return(shinyappsServerInfo())
-  }
+    info <- shinyappsServerInfo()
+  } else if (identical(name, cloudServerInfo(name)$name)) {
+    info <- cloudServerInfo(name)
+  } else {
+    configFile <- serverConfigFile(name)
+    if (!file.exists(configFile))
+      stop(missingServerErrorMessage(name))
```

Comment:
nit: braces always (existing code lacked them, but since this PR modified these lines ...)